.python-version

```diff
@@ -0,0 +1 @@
+3.5.2
```

Comment:
@dogweather I agree, but we need to see which version is supported in cloud.gov and also make sure Travis matches as well.  I'm running 3.5.2 locally right now (though 3.5.3 was just released) and have not run into any issues, but we need to ensure parity and support across all systems and environments we use.

I have asked to see what cloud.gov currently supports via the Cloud Foundry Python buildpack.  I know Travis supports 3.5+ too.  I do think it'd make more sense to make these changes in a separate PR, though, since there is a bit more work involved.